Dockerfile

```diff
@@ -60,18 +61,32 @@ RUN \
  tar xf \
 	/tmp/core.tar.gz -C \
 		/tmp/core --strip-components=1 && \
+ mkdir -p \
+		/tmp/base && \
+  if [ -z ${HASS_BASE+x} ]; then \
+	HASS_BASE=$(curl -sX GET https://api.github.com/repos/home-assistant/docker/releases/latest \
+	| jq -r .tag_name); \
+ fi && \
+ curl -o \
+	/tmp/base.tar.gz -L \
+		"https://github.com/home-assistant/docker/archive/${HASS_BASE}.tar.gz" && \
+ tar xf \
```

Comment:
There is no need to grab the tarball here. Pip install can do links, ie `pip install -r https://raw.githubusercontent.com/home-assistant/docker/master/requirements.txt`

Reply:
I thought you might want to lock the base image version with HASS_BASE. If not, that's definitely the better option.

Reply:
We can still do that.
`pip install -r https://raw.githubusercontent.com/home-assistant/docker/${HASS_BASE}/requirements.txt`

Reply:
Actually, now that I think about it it's probably better to grab that version from the HA repo itself so it's always kept in sync: https://github.com/home-assistant/core/blob/dev/build.json

Reply:
Just saw your second message, excellent. Best of both worlds. I'll make those changes.